plugin-viewport/src/main/kotlin/com/mapbox/maps/plugin/viewport/experimental/ViewportExt.kt

```diff
@@ -0,0 +1,13 @@
+@file:JvmName("ViewportUtils")
+
+package com.mapbox.maps.plugin.viewport.experimental
+
+import com.mapbox.maps.plugin.Plugin
+import com.mapbox.maps.plugin.delegates.MapPluginProviderDelegate
+
+/**
+ * Extension val for MapView to get the [ViewportPlugin] instance.
+ */
+val MapPluginProviderDelegate.experimentalViewport: ViewportPlugin
```

Comment:
maybe just `viewport`?

Reply:
For more context, this PR creates new package to prototype interfaces without the concrete implementation, and we will move these interfaces to the original package when the public interface is reviewed.

and experimentalViewport makes it easier to keep and test in parallel with the original implementation. It will be renamed to viewport once moved to the original package.